spine-godot/spine_godot/SpineSkeleton.h

```diff
@@ -66,6 +68,9 @@ class SpineSkeleton : public REFCOUNTED {
 	spine::Vector<float> bounds_vertex_buffer;
 	Ref<SpineSkin> last_skin;
 
+	std::unordered_map<spine::Bone*, Ref<SpineBone>> _cached_bones;
```

Comment:
When the `SpineSkeleton` is destroyed, will the `Ref`s to the bones and slots be decreased as well?